contracts/test/allowlist.test.ts

```diff
@@ -99,13 +99,4 @@ describe("AllowlistPaymaster", function () {
       expect(e.message).to.include("Ownable: caller is not the owner");
     }
   });
-
-  it("should prevent non-allowed user from calling Greeter", async function () {
```

Comment:
Why was this test removed? Is there a bug with `era-test-node`?

Reply:
Did not mean to remove. Will add back.

Reply:
Added back.